packages/EasyErrorHandler/src/Bridge/Symfony/Providers/ErrorCodesByEnumProvider.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyErrorHandler\Bridge\Symfony\Providers;
+
+use EonX\EasyErrorHandler\Interfaces\ErrorCodesEnumInterface;
+use EonX\EasyErrorHandler\Interfaces\ErrorCodesProviderInterface;
+use ReflectionClass;
+use Symfony\Component\Finder\Finder;
+
+final class ErrorCodesByEnumProvider implements ErrorCodesProviderInterface
+{
+    private const ERROR_CODE_NAME_PREFIX = 'Error';
```

Comment:
I believe this is some redundant. We may use all enum items.

Wdyt?

Reply:
Agreed, removed

Reply:
Thanks